src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,5 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_PROPERTY = new Prefix("pr/");
     public static final Prefix PREFIX_PREFERENCE = new Prefix("pf/");
-
+    public static final Prefix PREFIX_USERIMAGE = new Prefix("f/");
```

Comment:
Maybe change this to more intuitive letter? (i/ as image rather than f/ for file)

Reply:
Thanks for the suggestion. Prefix has been changed to `i/`